docs/philosophy/outcomes/Overview.md

```diff
@@ -69,6 +69,12 @@ Measures an organization’s ability to reduce waste by adopting common patterns
 
     Measures the sustainability of the platform and whether it continues to be used beyond initial onboarding
 
+-   :material-counter: __Adoption Density__
+
+    ---
+
+    Measures the quantity and quality of user activity
+
```

Comment:
I would change the order here to be the same as in `docs/philosophy/outcomes/developer_adoption.md`

here is
![Screenshot from 2023-09-29 10-34-33](https://github.com/dora-metrics/pelorus/assets/66965232/3d0c6ad0-3d66-4dc6-998b-8a7725198049)

in `docs/philosophy/outcomes/developer_adoption.md` is
![Screenshot from 2023-09-29 10-34-52](https://github.com/dora-metrics/pelorus/assets/66965232/3525ae78-ea88-443b-809c-24d958fa4823)

Reply:
Done.